src/net.h

```diff
@@ -951,11 +952,16 @@ class CNode
 
     void PushAddress(const CAddress& _addr, FastRandomContext &insecure_rand)
     {
+        // Whether the peer supports the address in `_addr`. For example,
+        // nodes that do not implement BIP155 cannot receive Tor v3 addresses
+        // because they require ADDRv2 (BIP155) encoding.
+        const bool supported = m_supports_addrv2 || _addr.IsAddrV1Compatible();
```

Comment:
Can you rename the var to something more meaningful? At least `addr_format_supported` maybe. I see the comment, but well, it won't hurt to have better war names too.